### DISCUSSION TITLE
Hold to confirm activation time should be independant from the pause action

### DISCUSSION THREAD

**Discussion by rodrigopina360** [ppy/osu#26846] [Ideas]
Currently people that are used to quick retrying maps are faced with a bit of an issue (especially ones with the quick retry keybind close to the keys used to play), as you have to either choose between having a delay that makes sure you don't press the key on accident, but makes pausing mid score very hard outside breaks, or be able to pause, but risk having the retry key be accidentally pressed, which can be an issue for people that have the quick retry key on some keypad.

My suggestion would be to make so that pausing is always instant (like it currently is with stable) so that hold to confirm activation time doesn't take effect on the pause button. 

Also another aside (didn't really feel like creating another discussion for such a simple question) but I've seen some work was put in place for the reintroduction of the Boss Key, is that something that's coming relatively soon, or if not, limited by people available to work on it? That's honestly a feature I'd really like to see, as I'm someone who uses it extensively on stable, and if I have a green flag, wouldn't mind giving it a try and putting work into implementing it from the bases built upon in https://github.com/ppy/osu-framework/pull/6085

**Comment by Rekunan** [comment]
The only exceptions I know to the setting are any "red" actions (deleting, removing, etc), so I'm not sure how I feel about making pause a part of those "red" actions.
However, I can empathize as a keypad user myself, and personally, I've elected to just rebinding that key to pause, since accidentally pausing is way better than accidentally quick-retrying, which may serve as a good solution for others, and maybe temporary if this does get planned to happen.

**Comment by qwuzzy** [comment]
Came here to make this suggestion myself. I don't like having to hold to confirm to pause a beatmap, but I also use the quick retry key as my main method to restart maps. I also use a keypad so it'd be great to not have quick retry be instantaneous incase of a missclick (which I've had happen.)
